Society/Society/Society/Components/Pages/MyProfile.razor

```diff
@@ -10,7 +10,7 @@
     </h1>
 </div>
 
-@if (userguid == null && user == null)
+@if (userguid == null && (user == null || user.Name == ""))
```

Comment:
IsNullOrEmpty(User)

Reply:
Kig if, else if tjek igennem. Jeg tror ikke der er behov for user.Name - det bedste ville nok være at vi arbjeder på enten user eller session

Reply:
user is not null